specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/datafactory.json

```diff
@@ -564,7 +651,7 @@
           "200": {
             "description": "OK.",
             "schema": {
-              "$ref": "#/definitions/IntegrationRuntimeStatusResponse"
+              "$ref": "#/definitions/IntegrationRuntimeResource"

```

Comment:
Here is to fix issue https://github.com/Azure/azure-rest-api-specs/issues/2057

Reply:
This fix should go into new API version since it is a breaking change. I'm adding it in my PR.

Reply:
Need to remove from here.